addon/components/accordion-list.js

```diff
@@ -13,11 +13,26 @@ const AccordionListComponent = Ember.Component.extend({
 
   // Internal state
   _activeItems: null,
+  _registeredItems: null,
   init() {
     this._super(...arguments);
     this.set('_activeItems', Ember.A([]));
+    this.set('_registeredItems', Ember.A([]));
   },
 
+  // If each registered item has at least on panel open, then return true
+  allExpanded: computed('_activeItems.[]', '_registeredItems.[]', function() {
+    let { _activeItems, _registeredItems } = this.getProperties('_activeItems', '_registeredItems');
+
+    for (let i=0; i < _registeredItems.length; i++) {
```

Comment:
An `array.reduce` might be cleaner here.

Reply:
good call. thanks.